lfs.c

```diff
@@ -3081,6 +3081,16 @@ static lfs_soff_t lfs_file_rawseek(lfs_t *lfs, lfs_file_t *file,
         return npos;
     }
 
+    int offset = npos - file->pos;                          // Get the difference between new and current file position
+    if (offset > 0) {                                       // If new position is after the current file position
```

Comment:
Small style nit: Could we reduce the column width to <=80 chars? Possibly just by putting the comments in front of this condition

Reply:
Hi geky,

as it is your project you could do what ever you want ;-).
Or does it mean, I should do the changes?
I don't want to go into a 'style' discussion. Just a little question: what is the reason for limiting the line length to 80?

Back to performance. As I wrote in issue #631: This change will only fix the case where the new position is after the old/previous position. It does not fix the other case, if the new position is before the old/previous position.
The problem is, that in this case, we do not know whether the data within the cache start from offset 0, or some bytes later. I think the first bytes of the cached block can contain some other date (block link data?).
This can only be fixed, if we would have an additional variable within the administrative structure which hold the start offset of the real file data?

BTW, I have an idea of an additional performance improvement, but I think it's better to open a new issue!?

Reply:
> > Just a little question: what is the reason for limiting the line length to 80?
> 
> At some point style is all just personal preference. I like keeping the line length to 80 so that I can fit several open files next to each other at once.

Depending on the project - a .editorconfig or a .clang-format can be of value. The former makes it easier to jump from project to project. The latter is more aggressive (performs automatic formatting for you) but can make format checks in a CI workflow trivial. I'm not necessarily suggesting either, but figured this information might be beneficial to some.

Reply:
I feel like the diversity of editors (which is a good thing) makes .editorconfig have limited value, but .clang-format is interesting to learn about. Though I'm hesitant to adopt automatic formatting as I think overzealous rule following does more harm than good, and automatic formatters tend to be really overzealous...

 Still, we really should have some simple CI rules for things like the 80 col rule, squiggly brackets, paren spacing, etc.